core/App/screens/ProofRequestAttributeDetails.tsx

```diff
@@ -60,13 +60,13 @@ const ProofRequestAttributeDetails: React.FC<ProofRequestAttributeDetailsProps>
       headerRight: undefined,
       headerLeft: () => (
         <HeaderLeftBack
-          title={t('Global.Back')}
+          title={' '}
```

Comment:
Was part of the ticket removing the back buttons?

Reply:
not officially, but the kim wanted the back button text removed

Reply:
In that case do we need the tile. Or will it display the default? If we need to use empty strings we should probably use have no spaces.

Reply:
it displays default without the title, but also when the title is set to an empty string for some reason. That's why I had to set it to be a space